application/server/server_test.go

```diff
@@ -520,6 +527,7 @@ func Test_initialize_integrationInClientInfo_readFromClientInfo(t *testing.T) {
 	currentConfig := config.CurrentConfig()
 	assert.Equal(t, expectedIntegrationName, currentConfig.IntegrationName())
 	assert.Equal(t, expectedIntegrationVersion, currentConfig.IntegrationVersion())
+	assert.Equal(t, expectedIdeVersion, currentConfig.IdeVersion())
```

Comment:
suggestion: maybe update the test to include IdeName and use a different value than IntegrationName